docs/core/howto/listings/trial/calculus/base_2.py

```diff
@@ -11,4 +11,4 @@ def multiply(self, a, b):
         return a * b
 
     def divide(self, a, b):
-        return a / b
+        return a // b
```

Comment:
One more thing.  Since you are doing division here, please add at the top of the file:

```from __future__ import division```

See: https://docs.python.org/2/library/__future__.html